htdocs/install/mysql/tables/llx_c_availability.sql

```diff
@@ -23,6 +23,8 @@ create table llx_c_availability
 	rowid		integer	 	AUTO_INCREMENT PRIMARY KEY,
 	code		varchar(30) NOT NULL,
 	label		varchar(128) NOT NULL,
+	type_duration	char(1),
```

Comment:
Do not use 'char', use always varchar, even for 1 char.

Reply:
i did it